.github/workflows/ci.yaml

```diff
@@ -106,13 +112,13 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Run cargo build with devnet-prealloc feature
-        run: cargo build --features devnet-prealloc --workspace --all --tests --benches
+        run: cargo build --release --features devnet-prealloc --workspace --all --tests --benches
         
       - name: Run cargo test regular features
-        run: cargo test --workspace
+        run: cargo nextest run --release --workspace
```

Comment:
Add doc test

Reply:
This should be covered by the `cargo test --release --doc --workspace --features=no-asm` I added